core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -629,49 +629,43 @@ class BrokerServer(
       .build()
   }
 
-  private def createShareCoordinator(): Option[ShareCoordinator] = {
-    if (config.shareGroupConfig.isShareGroupEnabled &&
-      config.shareGroupConfig.shareGroupPersisterClassName().nonEmpty) {
```

Comment:
So irrespective of feature flag or config the share-coordinator thread will run now. I think this is what you mentioned to be fixed in further PRs to start using feature listeners, correct?

Reply:
Correct.